src/code-templates/libraries/logger/logger.ts

```diff
@@ -1,10 +1,16 @@
-import { configurationProvider } from "configuration-provider";
+import {configurationProvider} from "../configuration-provider";
+import {pino} from "pino";
 
-// example on how to read the level: configurationProvider.get("logger.level")
-export const info = (message) => {
-  console.log(message);
-};
 
-export const error = (message) => {
-  console.error(message);
-};
+const logFile = configurationProvider.get("logger.destination");
+export default pino(
+    {
+        level: configurationProvider.get("logger.level"),
+        prettyPrint: {
```

Comment:
Great feature 💪

Reply:
I think so too, but I don't think it should be ON for all envs as JSONs are easier to work with for other logger services, so maybe only run this in case we're in development, WDYT?

Reply:
@henarbel Raz's advice sounds sensible to me, it's probably a dev artifact. I would apply it when a specific env var is set:

PRETTY_PRINT_LOG=true

We will set this for 'npm test', 'start:dev'

I believe that relying on ENV=prod|dev is fragile, it encourages developers to code different things for dev-vs-prod.

Reply:
OK as requested,  I've added some level of decoupling between the code and pino using a simple interface for `Logger` which is wrapping the pino instance and accepts the necessary configurations.
I know we've talked about it but while implementing I thought it makes sense to decouple also from `configuration-provider` because its easier to test/extend. if you still think its important I can change it. 
in the end the DX stays the same but the implementation is a bit more flexible.